packages/react-storage/src/components/StorageBrowser/actions/types.ts

```diff
@@ -20,6 +21,8 @@ export interface TaskData {
   id: string;
 }
 
+export interface ActionData extends TaskData, FileData {}
```

Comment:
Not all actions use this shape, this needs to be removed in a follow up

Reply:
Yeah, this is just a hack to get the types working, will follow up on the right way to do this